frontend/src/views/merchantsList.vue

```diff
@@ -1,59 +1,168 @@
-      class="card"
-      v-bind:key="merchant.key"
-    >
-      <div class="card-content">
-        <div class="media">
-          <div class="media-content">
-            <p class="title is-4 has-text-info is-size-2.5">
-              {{merchant.merchantName}}
-              <span class="is-pulled-right">{{merchant.totalPrice}} ₹</span>
-            </p>
+  <div>
+    <Logout />
+    <div class="container">
+      <div
+        @click="ItemDetails(merchant)"
+        v-for="merchant in merchants"
+        class="card"
+        v-bind:key="merchant.key"
+      >
+        <div class="card-content">
+          <div class="media">
+            <div class="media-content">
+              <p class="title is-4 has-text-info is-size-2.5">
+                {{merchant.merchantName}}
+                <span class="is-pulled-right">{{merchant.totalPrice}} ₹</span>
+              </p>
+            </div>
           </div>
+          <div
+            class="has-text-grey-light"
+            v-for="item in merchant.itemDetails"
+            v-bind:key="item.key"
+          >
+            <span v-if="!item.isAvailable">{{item.merchantItemName}} not available</span>
+          </div>
+          <p class="subtitle is-6 is-pulled-left">
+            Delivery in
+            <span class="has-text-info">{{timeString}}</span>
+          </p>
         </div>
-        <div class="has-text-grey-light" v-for="item in merchant.itemDetails" v-bind:key="item.key">
-          <span v-if="!item.isAvailable">{{item.merchantItemName}} not available</span>
-        </div>
-        <p class="subtitle is-6 is-pulled-left">Delivery in <span class="has-text-info">{{timeString}}</span></p>
+      </div>
+      <div id="delivery" class="card">
+        <p>Click on merchant to get more details and placing the order</p>
       </div>
     </div>
-    <div id="delivery" class="card">
-      <p>Click on merchant to get more details and placing the order</p>
-    </div>
-  </div>
+    <gmap-map :center="center" :zoom="17" style="width:100%;  height: 400px;">
+      <gmap-marker
+        :key="index"
+        v-for="(m, index) in markers"
+        :position="m.position"
+        icon="http://maps.google.com/mapfiles/ms/icons/green-dot.png"
+      />
+
+      <gmap-marker
+        :key="index2"
+        v-for="(m, index2) in markers2"
+        :position="m.position"
+        icon="http://maps.google.com/mapfiles/ms/icons/orange-dot.png"
+      />
+    </gmap-map>
   </div>
 </template>
 
 <script>
 import firebase from "firebase";
 import Logout from "@/components/Logout.vue";
+import * as Geofire from "geofire";
+var j;
 
 export var merchantexp;
 export var itemexp;
 export var time;
 
 export default {
   name: "merchantList",
-  components:{
+  components: {
     Logout
   },
   data() {
     return {
-      merchants: [],timeString:""
+      merchants: [],
+      timeString: "",
+      center: { lat: 45.508, lng: -73.587 },
+      markers: [],
+      markers2: [],
```

Comment:
Can you please change marker2 to something meaningful like, merchantLocationMarkers